docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>Can we add one sentence after this cell to say that something along the line of 9 CNOTs is better than 19 CNOTs. It might not be obvious to beginners.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>